standard.scad

```diff
@@ -69,6 +69,8 @@ bp_rcut_width = 8.5;
 bp_rcut_length = 4.25;
 // Baseplate bottom cutout rounded thingy depth
 bp_rcut_depth = 2;
+// Baseplate clearence offset
+bp_xy_clearence_offset = 0.5;
```

Comment:
English is not my first language. Does the name need to be this long or would 'bp_xy_clearance' be enough?

Reply:
that should be fine, give me a moment